src/routes/open/components/Layout.test.js

```diff
@@ -66,16 +66,13 @@ describe('React DOM TESTS > Create Safe form', () => {
     await sleep(1500)
 
     // THEN
-    const Deployed = TestUtils.findRenderedDOMComponentWithClass(open, DEPLOYED_COMPONENT_ID)
```

Comment:
@germartinez Just refactoring this test because Flow found some errors. You can skip it.